app/pods/publications/publication/publications/route.js

```diff
@@ -0,0 +1,16 @@
+import Route from '@ember/routing/route';
+
+export default class PublicationsRoute extends Route {
+  model() {
+    return this.modelFor('publications.publication');
+  }
+
+  async afterModel() {
+    this.publicationSubcase = await this.modelFor('publications.publication').publicationSubcase;
+  }
```

Comment:
Ik zou dit eerder omgekeerd verwachten, dat `publicationSubcase` het hoofdmodel is en de `publicationFlow` een side model dat gezet wordt in de `afterModel`-hook. Zo is het ook in de `publications/publication/translations` route.